src/views/ProfileView.vue

```diff
@@ -0,0 +1,362 @@
+              <v-simple-table>
+                <template v-slot:default>
+                  <tbody>
+                    <tr
+                      v-for="item in schuleData.labelAndValues"
+                      :key="item.label"
+                    >
+                      <td class="padding">
+                        <span v-if="item.labelAbbr"
+                          ><abbr :title="item.label"
+                            ><strong>{{ item.labelAbbr }}</strong></abbr
+                          >:</span
+                        >
+                        <strong v-else>{{ item.label }}:</strong>
+                      </td>
+                      <td>{{ item.value }}</td>
+                    </tr>
+                  </tbody>
+                </template>
+              </v-simple-table>
+              <p
+                v-if="schuleData.schoolAdmins.length > 0"
+                class="info"
+              >
+                <v-icon
+                  class="mr-2"
+                  icon="mdi-information-slab-circle-outline"
+                ></v-icon>
+                {{ schuleData.info + ' ' + schuleData.schoolAdmins?.join(', ') }}
+              </p>
+            </v-col>
+          </v-row>
+        </LayoutCard>
+      </v-col>
+      <v-col
+        cols="12"
+        sm="12"
+        md="6"
+      >
+        <LayoutCard :header="$t('login.password')">
+          <v-row class="ma-3 d-flex align-content-center justify-center ga-4">
+            <v-icon
+              size="x-large"
+              class="full-width"
+              icon="mdi-key-alert-outline"
+            ></v-icon>
+            <div>
+              <v-btn
+                color="primary"
+                disabled
+              >
+                Passwort ändern
+              </v-btn>
+            </div>
+          </v-row>
+        </LayoutCard>
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="12"
+        md="6"
+      >
+        <LayoutCard :header="$t('profile.twoFactorAuth')">
+          <v-row class="ma-3 d-flex align-content-center justify-center ga-4">
+            <v-icon
+              size="x-large"
+              class="full-width"
+              icon="mdi-shield-account-outline"
+            ></v-icon>
+            <div>
+              <v-btn
+                color="primary"
+                disabled
+              >
+                2-FA einrichten
+              </v-btn>
+            </div>
+          </v-row>
+        </LayoutCard>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<style>
+  .profile {
+    margin: 20px;
+  }
+
+  .headline {
+    margin-bottom: 20px;
+  }
+
+  .right {
+    text-align: right;
+  }
+
+  .center {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    gap: 5px;
+  }
+
+  .padding {
+    padding: 8px;
+  }
+
+  .full-width {
+    width: 100%;
+  }
+
+  .info {
+    padding-top: 20px;
+    text-align: center;
+    color: grey;
+  }
+</style>
```

Comment:
Generell können viele dieser Stile durch Vuetify Bordmittel ersetzt werden. Zb die Spacings: https://vuetifyjs.com/en/styles/spacing/#how-it-works

Sollte es um einzelne Pixel gehen, bevorzugen wir die Abstände, die Vuetify bereitstellt, das ist mit dem Designer mal abgesprochen worden.